Jenkinsfile

```diff
@@ -44,4 +44,4 @@
 packageBuildingPipelineDAOSTest(['distros': ['el8', 'centos7', 'leap15', 'ubuntu20.04'],
                                  'make args': 'DISTRO_VERSION_EL8=8.5',
                                  'rpmlint_rpms_allow_errors': true,
-                                 'test-tag': 'daosio'])
+                                 'test-tag': 'pr'])
```

Comment:
I was not entirely sure of what should be used, it sounded like `daosio` was not the right tag, @frostedcmos suggested to use `pr`, I'm just trying to target something that is not too heavy in terms of testing but still covers enough stuff. To be fair we should create a special mercury tag but that needs to be defined too.

Reply:
Indeed, I am not entirely sure either.  I am just skeptical that `pr` is the right choice.  Frankly I don't think `pr` is probably the right choice for just about any PR going into CI.

Historically the set of tests were tagged `pr` was at one time actually stripped back by quite a bit due to the load the `pr` set was putting on our clusters at the time.  When it was stripped back, it was also expected that developers would add `Features:` tags to choose a group of tests that would target the code changes in their PRs.  Unfortunately I don't think too many developers actually do this.  There is work underway to try to somewhat heuristically suggest tags at commit time so hopefully things will get better there.

So, ideally here, want a set of test tags that specifically exercises mercury, which I guess means tests that focus on network IO (realizing of course that anything to do with DAOS involves network IO), and perhaps stress tests it and/or pushes into the corner cases of network IO, etc.

I don't know which tags that is, and as you suggest it would be useful to curate a set of tests and tag them mercury.  Indeed, I have suggested in the past that this be done for every one of our dependencies so that we can imply the dependency name into the test tag to be used.

Maybe a query to the Functional test engineers about which tests would specifically focus on networking and mercury might get you some input.  I don't know what the `daosio` tests are but the tag name sounds good, in terms of targeting network IO.  I suppose that could be disk IO also though.  :smirk: 

It looks like `daosio` was added by @jgmoore-or in https://github.com/daos-stack/mercury/commit/8c5dc136833322c8eff565df0caba28e314d8867.  Maybe he has some insight into choosing that tag specifically.

Reply:
I reverted back to `daosio`.